src/Service.Host/client/src/selectors/serialNumberTransactionSelectors.js

```diff
@@ -1,4 +1,50 @@
-﻿import ItemSelectors from './ItemSelectors';
-import * as itemTypes from '../itemTypes';
+﻿import * as itemTypes from '../itemTypes';
 
-export default new ItemSelectors(itemTypes.serialNumberTransaction.item);
+export default new function serialNumberTransaction() {
+    const { item } = itemTypes.serialNumberTransaction;
+
+    this.getItem = state => {
+        const storeItem = state[item];
+        if (!storeItem) {
```

Comment:
As with actions - could this use the standard selectors?

Reply:
i tried, unsuccessfully, to mutate the generic function.. will have another look